templates/app/client/components/auth(auth)/auth.service.js

```diff
@@ -161,8 +161,10 @@ export function AuthService($location, $http, $cookies, $q, appConfig, Util, Use
     hasRole(role, callback?: Function) {
       return Auth.getCurrentUser(undefined)
         .then(user => {
-          var has = user.hasOwnProperty('role')
+          var has = user
+            ? user.hasOwnProperty('role')
             ? hasRole(user.role, role)
+            : false
```

Comment:
This makes more sense to me:

```js
let has = user && user.hasOwnProperty('role') && hasRole(user.role, role);
```

I believe this will result in the same output.